java/simplesaga-dsl/src/main/java/io/simplesource/saga/dsl/SagaDsl.java

```diff
@@ -1,4 +1,4 @@
-package io.simplesource.saga.saga.dsl;
```

Comment:
.The dsl is client only:
Maybe we should move all the client stuff into here (including `KafkaSagaAPI`).
And then it might make sense to name the module `simplesaga-client`.
We can also do this in a follow up refactoring.
Or would that be awkward from a dependency point of view?

Reply:
OK, I'll do that in follow up refactoring.